doc/sphinx/dials_scale_user_guide.rst

```diff
@@ -2,11 +2,13 @@ User guide for scaling data with DIALS
 ======================================
 
 This document aims to provide a guide to using :samp:`dials.scale`, at various levels
-of depth. A new user is encouraged to read the symmetry and scaling section of
-the 'Processing in detail' tutorial for a quick overview of scaling in DIALS.
-For most users, it is likely to be sufficient to read only the 'Byte-sized guide
-to common scaling options' below, and return to the rest of the guide if further
-help is needed.
+of depth. A new user is encouraged to read the Symmetry and Scaling sections of
+the `Processing in detail
+<https://dials.github.io/documentation/tutorials/processing_in_detail_betalactamase.html>`_
```

Comment:
Can you link directly to the Symmetry and Scaling sections here? I think you should also be able to use the built in [sphinx/rst cross-referencing mechanism](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations) rather than hardcoding the URL as you have done here.